marklogic-data-hub/src/test/java/com/marklogic/hub/flow/VerifyBatchTimestampTest.java

```diff
@@ -0,0 +1,28 @@
+package com.marklogic.hub.flow;
+
+import com.marklogic.client.io.Format;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.step.RunStepResponse;
+import com.marklogic.hub.test.Customer;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class VerifyBatchTimestampTest extends AbstractHubCoreTest {
+
+    @Test
+    void test() {
+        installProjectInFolder("test-projects/custom-step-with-sleep");
+        
+        // writeManyCustomers(customerCount);
+        RunFlowResponse response = runFlow(new FlowInputs("customFlowWithSleep"));
+        RunStepResponse stepResponse = response.getStepResponses().get("1");
+
+        assertEquals("completed step 1", stepResponse.getStatus());
+        assertNotEquals(stepResponse.getStepStartTime(), stepResponse.getStepEndTime());
```

Comment:
I think we want 3 assertions here:

1. The start time is not null
2. The end time is not null
3. The end time is after the start time 

To perform assertion 3, you'll need to get each time as a LocalDateTime. That would be a useful method to add to RunStepResponse. 

Unfortunately, we don't seem to have any good helper methods for parsing these times using the more reliable DateTimeFormatter in Java 8 (as opposed to SimpleDateFormat). I just whipped up this quick test to verify how to construct a DateTimeFormatter that works on ML timestamps, which can have variable lengths of microseconds. You can use this code in your test to verify that the end time is after the start time - no need to add to RunStepResponse for now, I'll take care of that in a future PR.

```
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
import java.time.format.DateTimeFormatterBuilder;
import java.time.temporal.ChronoField;

public class MyTest {

    public static void main(String[] args) {
        String before = "2021-02-18T14:28:26.123456Z";
        String after = "2021-02-18T14:28:27.526055Z";

        DateTimeFormatter formatter = new DateTimeFormatterBuilder()
            .appendPattern("yyyy-MM-dd'T'HH:mm:ss")
            .appendFraction(ChronoField.MICRO_OF_SECOND, 0, 6, true)
            .appendPattern("'Z'")
            .parseLenient()
            .toFormatter();
        LocalDateTime beforeTime = LocalDateTime.parse(before, formatter);
        LocalDateTime afterTime = LocalDateTime.parse(after, formatter);
        System.out.println(beforeTime);
        System.out.println(afterTime);
        System.out.println(afterTime.isAfter(beforeTime));
    }
}
```